packages/angular_devkit/build_webpack/src/webpack/index.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  BuildEvent,
+  Builder,
+  BuilderConfiguration,
+  BuilderContext,
+} from '@angular-devkit/architect';
+import { Path, getSystemPath, normalize, resolve } from '@angular-devkit/core';
+import { Observable } from 'rxjs';
+import * as webpack from 'webpack';
+import { WebpackBuilderSchema } from './schema';
+
+export class BrowserBuilder implements Builder<WebpackBuilderSchema> {
+
+  constructor(public context: BuilderContext) { }
+
+  run(builderConfig: BuilderConfiguration<WebpackBuilderSchema>): Observable<BuildEvent> {
```

Comment:
I kinda feel like making it a base class would end up being more lines to maintain. The only reason why I didn't put the one line `loadWebpackConfig` inside run was because I thought other users would want to load the config from the builder.

Do you have a suggestion that you feel would be better?

Reply:
It's fine.  The code count is minimal.  Whenever I see repeat code, I like to question whether there is an opportunity for a refactor.